imgui_plot.h

```diff
@@ -1062,18 +1046,18 @@ namespace ImGui
 
                 // start: end of the outer arc.  end: start of the inner arc
                 const ImVec2 vec1 = ImVec2(
-                    ImCos(x_data_rad)*properties_.inner_radius + centre.x, 
+                    ImCos(x_data_rad)*properties_.inner_radius + centre.x,
                     ImSin(x_data_rad)*properties_.inner_radius + centre.y);
 
                 // start: end of the inner arc.  end: start of the outer arc
                 const ImVec2 vec2 = ImVec2(
-                    ImCos(x_data_prev_rad)*properties_.outer_radius + centre.x, 
+                    ImCos(x_data_prev_rad)*properties_.outer_radius + centre.x,
                     ImSin(x_data_prev_rad)*properties_.outer_radius + centre.y);
 
                 // draw the pie segment
                 window->DrawList->PathArcTo(centre, properties_.outer_radius, x_data_prev_rad, x_data_rad, n_segments);  // outer arc
                 window->DrawList->PathLineTo(vec1);  // start outer to inner arc line
-                window->DrawList->PathArcTo(centre, properties_.inner_radius, x_data_rad, x_data_prev_rad, n_segments);  // inner arc 
+                window->DrawList->PathArcTo(centre, properties_.inner_radius, x_data_rad, x_data_prev_rad, n_segments);  // inner arc
                 window->DrawList->PathLineTo(vec2);  // end outer to inner arc line
                 window->DrawList->PathFillConvex(colors[n]);
```

Comment:
Thanks for the updates above.

Reply:
The regular expression I use for this is `" +$"` (without the double quotes). This removes all the spaces at the end of lines.

Reply:
That is cool.  I will for sure be using that!